google/cloud/security/inventory/inventory_loader.py

```diff
@@ -64,6 +65,7 @@
 from google.cloud.security.inventory.pipelines import load_orgs_pipeline
 from google.cloud.security.inventory.pipelines import load_projects_iam_policies_pipeline
 from google.cloud.security.inventory.pipelines import load_projects_pipeline
+from google.cloud.security.inventory.pipelines import load_projects_buckets_pipeline
```

Comment:
nit: I think that this import should be sorted above the import for load_projects_iam_policies_pipeline